### DISCUSSION TITLE
Deployment issues

### DISCUSSION THREAD

**Discussion by Yash-finwiz** [devfinwiz/Fin-Maestro#49] [General]
can you help deploy this project Because I am getting runtime error, frontend is working but backend is not working.

**Comment by devfinwiz** [comment]
@YASH260 Can you elaborate on what the error is and in which module? Screenshot spam would work too.

**Comment by Yash-finwiz** [comment]
[Runtime_Error.pdf](https://github.com/devfinwiz/Fin-Maestro-Web/files/13793597/Runtime_Error.pdf)
I have uploaded the PDF document, which includes screenshots and details of a runtime error. I attempted to install the project as outlined in the Readme.md file, but unfortunately, none of them seem to be functioning correctly. I would greatly appreciate your assistance in resolving this matter. Thank you for your time and support.

**Reply by devfinwiz** [reply]
Hello @YASH260 , 
Appreciate your efforts to put this all up together. The front end (Vue.js) and the backend are not on the same track I guess since there are multiple problems with YahooFinancials API. 

One of the solutions can be to append  . NS at the end of the symbol if you're not choosing from the suggestions list. 

Another solution can be to use the CommonFEGradio from the master branch to bring the UI up for key modules. 

Apologies for inconveniences since I am unable to find time to contribute (maintain) to my old open source projects.
